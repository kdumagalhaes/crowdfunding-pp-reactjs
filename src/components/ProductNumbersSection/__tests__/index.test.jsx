import { render } from "@testing-library/react";
import { ProductNumbersSection } from "..";

describe("ProductNumbersSection", () => {
  it("initial component should match snapshot", () => {
    const { asFragment } = render(<ProductNumbersSection />);
    expect(asFragment()).toMatchSnapshot();
  });
});
