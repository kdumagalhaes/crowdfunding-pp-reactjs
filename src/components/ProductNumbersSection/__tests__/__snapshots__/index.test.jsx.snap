// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductNumbersSection initial component should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-bdvvtL jnOboy"
  >
    <section
      class="sc-gsDKAQ esfoTq"
    >
      <div
        class="flex-container"
      >
        <div
          class="number"
        >
          <span
            class="number-value"
          >
            $89,914
          </span>
          <span
            class="number-backed"
          >
            of $100,000 backed
          </span>
        </div>
        <div
          class="number"
        >
          <span
            class="number-value"
          >
            5,007
          </span>
          <span
            class="number-backed"
          >
            total backers
          </span>
        </div>
        <div
          class="number"
        >
          <span
            class="number-value"
          >
            56
          </span>
          <span
            class="number-backed"
          >
            days left
          </span>
        </div>
      </div>
      <input
        class="value-range"
        max="100000"
        min="0"
        type="range"
        value="89914"
      />
    </section>
  </div>
</DocumentFragment>
`;
